package.json

```diff
@@ -33,6 +33,7 @@
         "@types/jest": "^27.4.1",
         "@types/node": "^17.0.27",
         "@types/react": "^18.0.7",
+        "@types/react-beautiful-dnd": "^13.1.8",
```

Comment:
Will it be better if we add this lib to trader package.json?
Please, ignore if irrelevant

Reply:
yes. should be in trader.

Reply:
I will check, currently adding this package somehow leads to removal of 20k lines from package-lock as well 🫠

Reply:
Add both packages to trader and run npm run bootstrap:dev. Should be ok

Reply:
guys please never add anything to the main package.json. it kills package-lock )) 
every time analytics package is updated to the next version, guys are literally suffering.

Reply:
Hmm, that's a nice explanation @maryia-deriv, I'm just wondering, WHY do we not want to update it if there's `^` in the version. That is literally the purpose of adding this symbol in the package version. I didn't get to add a new package to the repo for a long time, but clearly something is very wrong with the way package-lock is utilized.

Reply:
removal of the caret is being discussed